doc/proposals/scorecard-security.md

```diff
@@ -0,0 +1,87 @@
+---
+title: Testing the Security of Operators by tooling
+authors:
+  - "@camilamacedo"
+reviewers:
+  - TBD
+approvers:
+  - TBD
+creation-date: 2019-10-31
+last-updated: 2019-10-31
+status: provisional
+see-also:
+  - "doc/proposals/openshift-4.3/operator-testing-tool.md" 
+---
+
+# Testing the Security of Operators by tooling
+
+## Release Signoff Checklist
+
+- \[ X\] Enhancement is `provisional`
+- \[ X\] Design details are appropriately documented from clear requirements
+- \[ \] Test plan is defined
+- \[ \] Graduation criteria for dev preview, tech preview, GA
+- \[ \] User-facing documentation is created in [operator-sdk/doc][operator-sdk-doc]
+
+## Open Questions (optional)
+
+- Would have more security checks which may are important for the Operators and are not mapped here? 
+
+## Summary
+
+Currently, the scorecard will perform some tests. For further information see its [doc](https://github.com/operator-framework/operator-sdk/blob/master/doc/test-framework/scorecard.md#tests-performed).  
+This proposal is regarded to add a new suite of tests in order to ensure the quality of the operators by checking if they are complying with few common/general security practices. 
+In order to clarifies and exemplify this need as solution see the project https://kubesec.io/ and its demo which verifies and check if the resource is complying with the security common practices.   
+
+## Motivation
+
+- Ensure the quality of the operators projects by checking their security.   
+- Ensure the quality by checking the security of the projects which are distributed by OperatorHub.io and OpenShift catalogue
+
+## Goals
+
+* The test should fail when 1 or more resources created by the project are not complaining with the security rules adopted
+
+## Proposal
+
+### User Stories (optional)
+
+#### Story 1
+
+I as an operator dev user, I'd like to check the quality of my operator by verifying via tooling if it is complying with the minimal/common requirements. 
+
+**Acceptance Criteria**
+
+- The tooling/check should fail if the operator project creates a resource which not complain with the roles defined
+- The tooling/check should fail if the operator project do not label all resources created by it
+- The tooling/check should show what/where/why the operator did not pass in the tests for each failure rule
+
+### Implementation Details/Notes/Constraints
+
+For the following checks is possible to verify the Operator.YAML file:
```

Comment:
Are we sure all the tests / rules below can be addressed/ checked with static tests?

Reply:
My idea here is to parse the yaml to JSON and check if these fields are populated or not. Shows quite simple. 
Are you able to see some pitful on that that I could not?

Reply:
Info added in the proposal as well so I am closing it.